test_extras/test_chemprop/test_models.py

```diff
@@ -177,3 +192,60 @@ def test_get_params(self) -> None:
                 self.assertEqual(
                     param_dict[param_name], param, f"Test failed for {param_name}"
                 )
+
+class TestChempropMulticlassClassifier(unittest.TestCase):
+    """Test the Chemprop classifier model."""
+
+    def test_get_params(self) -> None:
+        """Test the get_params and set_params methods."""
+        n_classes = 3
+        chemprop_model = ChempropMulticlassClassifier(lightning_trainer__accelerator="cpu", n_classes=n_classes)
+        param_dict = chemprop_model.get_params(deep=True)
+        expected_params = dict(DEFAULT_MULTICLASS_CLASSIFICATION_PARAMS)  # Shallow copy
+        expected_params["model__predictor__n_classes"] = n_classes
+        expected_params["n_classes"] = n_classes
+        self.assertSetEqual(set(param_dict.keys()), set(expected_params.keys()))
```

Comment:
use `compare_params` from chemprop_test_utils/compare_models.py, see example [here](https://github.com/basf/MolPipeline/blob/main/test_extras/test_chemprop/test_neural_fingerprint.py#L28)

Reply:
also needs to be adapted in other tests

Reply:
This does not work as we compare dicts and not models.